src/app/bungie-api/http-client.ts

```diff
@@ -146,7 +148,7 @@ export function createHttpClient(
       for (const key in config.params) {
         typeof config.params[key] === 'undefined' && delete config.params[key];
       }
-      url = `${url}?${new URLSearchParams(config.params).toString()}`;
+      url = `${url}?${new URLSearchParams(config.params as Record<string, string>).toString()}`;
```

Comment:
if i can remember how to supplement it, i have a more accurate type for `URLSearchParams`'s constructor